2018-04-18.md

```diff
@@ -0,0 +1,26 @@
+---
+to: All Teams
+subject: New SourceBots Kit Update
+---
+
+Hello!
+
+As the competition draws closer, it's time to release what is likely the final kit update before the competition itself. This release is a small one, but it's important you have updated to it before the competition.
+
+# Changes
+The largest change is the setting of your starting zone and mode using an additional USB drive. By default, your robot is in zone 0, and in development mode. With these USB drives, it sets your robot into competition mode, and correctly sets the starting zone for you when your match starts.
+
+Access to the `zone` and `mode` is now correctly documented: https://docs.sourcebots.co./api/game-state/.
```

Comment:
don't bother implying that it existed before, I would just say it was added (to reduce confusion)